Batteries/Control/OptionT.lean

```diff
@@ -47,6 +47,8 @@ variable [Monad m]
 
 @[simp] theorem run_pure (a) : (pure a : OptionT m α).run = pure (some a) := rfl
 
+/-- This was changed during porting from mathlib to use Option.elimM rather than explicit
+pattern matching, aligning with `run_seq` and `run_orElse`. -/
```

Comment:
This belongs in the PR description, not the file.

Reply:
Ah sorry, that makes sense